.github/workflows/macstadium-android-e2e.yml

```diff
@@ -9,7 +9,7 @@ on: [pull_request, workflow_dispatch]
 jobs:
   # This workflow contains a single job called "android-e2e"
   android-e2e:
-    if: github.ref_name == '@tchayen/android-detox'
+    if: github.head_ref == '@tchayen/android-detox'
```

Comment:
do we want to start e2e that are working on android on develop as well?

Reply:
no, we still want ot reuse this branch name to restrict  the impact

Reply:
As Michał said, the stability of those is still questionable and probably we don't want to risk enabling them yet.